argo/argo-cluster-generation-from-rancher-capi/argo-cluster-generation-from-rancher-capi.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: argo-cluster-generation-from-rancher-capi
+  annotations:
+    policies.kyverno.io/title: Argo Cluster Secret Generation From Rancher CAPI Secret
+    policies.kyverno.io/category: Argo
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Cluster
+    kyverno.io/kyverno-version: 1.7.0
```

Comment:
Was this tested with 1.7.0 or 1.7.1? This annotation's value should reflect the version of Kyverno you actually used to verify its operation.